language/e2e-tests/src/tests/validator_set_management.rs

```diff
@@ -24,11 +24,11 @@ fn validator_add() {
 
     let txn = register_validator_txn(
         new_validator.account(),
+        vec![255; 32],
+        vec![254; 32],
+        vec![253; 32],
         vec![],
-        vec![],
-        vec![],
-        vec![],
-        vec![],
+        vec![252; 32],
```

Comment:
what're these magic numbers..

Reply:
Keys are 32-bytes, if they are not, we fail a check when reading the validator set from storage. They also have to be of a certain size, if they are not, they will fail a ed25519 check during deserialization.